### ISSUE TITLE
Question: Always Encrypted Support

### ISSUE THREAD

**Issue by PA-LynnPettis** [michiya/django-pyodbc-azure#178]
Does django-pyodbc-azure support always encrypted with SQL Server 2016 and SQL Server 2017 servers?

**Comment by rogress** [comment]
We are about to find out this weekend, we are going to test it.  I'll report back.

**Comment by PaszaVonPomiot** [comment]
Yes. We use always encrypted connection settings on SQL Server 2016 and 2017.

**Comment by nyashaChiza** [comment]
l am connecting to the sql server using ssms ad it only allows a connection if l set the option encrypt connection to false, l want to connect using django-pyodbc-azure , how do l set the encrypt connection to false, l tried several ways ad l keep getting the error:
django.db.utils.InterfaceError: ('28000', "[28000] [Microsoft][ODBC Driver 17 for SQL Server][SQL Server]Login failed for user 'user'.
(18456) (SQLDriverConnect); [28000] [Microsoft][ODBC Driver 17 for SQL Server]Invalid connection string attribute (0); [28000] [Microsoft][ODBC Driver 17 for SQL Server][SQL Server]Login failed for user 'user'. (18456); [28000] [Microsoft][ODBC Driver 17 for SQL Server]Invalid connection string attribute (0)")
